lib/shortcuts/stream-lambda.js

```diff
@@ -59,6 +61,28 @@ class StreamLambda extends Lambda {
         StartingPosition
       }
     };
+    if (FilterCriteria) {
+      if (Object.prototype.toString.call(FilterCriteria) !== '[object Object]'){
```

Comment:
I wonder if we can remove all this code in favor of using something like https://github.com/hapijs/joi

Reply:
That would likely be cleaner, but I think it would be more appropriate to have that in a separate refactor that applies JOI validation for all shortcuts